### ISSUE TITLE
[Bug]: test_lldp_neighbor_post_orchagent_reboot fails for multi-asic

### ISSUE THREAD

**Issue by anamehra** [sonic-net/sonic-mgmt#14689]
### Issue Description

https://github.com/sonic-net/sonic-mgmt/pull/14616 introduced new test test_lldp_neighbor_post_orchagent_reboot
The test case does not handle multi-asic scenario and fails for multi-asic DUT

### Results you see

On multi-asic platform, the tests try to disable swss0 feature. feature names should not be appended with namespace number. 

>           raise RunAnsibleModuleFail("run module {} failed".format(self.module_name), res)
E           tests.common.errors.RunAnsibleModuleFail: run module shell failed, Ansible Results =>
E           {"changed": true, "cmd": "sudo config feature autorestart swss0 disabled", "delta": "0:00:01.131598", "end": "2024-09-22 09:24:54.296006", "failed": true, "msg": "non-zero return code", "rc": 1, "start": "2024-09-22 09:24:53.164408", "stderr": "", "stderr_lines": [], "stdout": "Feature 'swss0' doesn't exist", "stdout_lines": ["Feature 'swss0' doesn't exist"]}

### Results you expected to see

The test should pass on multi-asic as well

### Is it platform specific

generic

**Comment by anamehra** [comment]
Hi @abdosi , for you viz.

**Comment by arlakshm** [comment]
New test added, does not handle multi-asic.

**Comment by arlakshm** [comment]
Another PR  #13172  in flight for fixing the lldp failures on multi-asic devices for existing test. @anamehra  to fix this new test after #13172 is merged.

**Comment by anamehra** [comment]
looks like recent changes in lldp script handles this scenario.

**Comment by anamehra** [comment]
needs fix.
